edx_proctoring/templates/proctoring/seq_proctored_exam_footer.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+<div class="faq-proctoring-exam">
+  <h4> {% trans "See Also" %} </h4>
+    <p>
```

Comment:
Per discussion with @griffresch we would like to use the same 3 See Also links at the bottom of every template page, for simplicity/consistency.

- About proctored exams
- Online proctoring rules
- Technical requirements for taking a proctored exam

- "About proctored exams" goes to http://edx.readthedocs.org/projects/edx-guide-for-students/en/latest/sfd_proctoredexams/SFD_ProctoredExams.html
- "Online proctoring rules" goes to: http://edx.readthedocs.org/projects/edx-guide-for-students/en/latest/sfd_proctoredexams/SFD_OnlineProctoringPolicy.html
- "Technical requirements" goes to: http://clientportal.softwaresecure.com/support/index.php?/Knowledgebase/Article/View/252/0/system-requirements-remote-proctor-now